src/core_plugins/metrics/public/components/panel_config/markdown.js

```diff
@@ -124,7 +124,7 @@ class MarkdownPanelConfig extends Component {
             <div className="vis_editor__label">Custom CSS (supports Less)</div>
           </div>
           <div className="vis_editor__ace-editor">
-            <AceEditor
+            <KuiCodeEditor
               mode="less"
               theme="github"
```

Comment:
Are there use-cases for specifying other `theme`s than `github`? If not we should pull it into `KuiCodeEditor`. I'd probably rename `mode` to `language` too.

Reply:
This gets back to the question on what level this abstraction should be. In the long term I would agree with you, and rename those, but for now I would just pass them through, since we are currently passing all properties just to the editor, and I think if we start renaming we should make all properties explicit again (which for this first version we decided against).